### PR TITLE
chore: migrate to owl bot

### PR DESCRIPTION
**Pull request by parthea**

### CODE REVIEW

File: owlbot.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Copyright 2018 Google LLC
```
Comment (busunkim96) [inline owlbot.py:1]: Huh I wonder why git didn't think this was a file rename like in other PRs.

**Review by busunkim96 — APPROVED** [review]

File: owlbot.py (hunk 1 above)
Comment (parthea) [reply owlbot.py:1]: Let me retry the migration and see if that fixes it.

Comment (parthea) [reply owlbot.py:1]: I tried renaming the file using `git mv` but the `synth.py` file still shows as deleted and `owlbot.py` added in github. My guess is that when the changes are significant github can't display the renaming properly.

**Comment by parthea** [comment]
I'm going to force push this branch due to the conflicting files

**Comment by parthea** [comment]
I'm going to force push this branch again due to issue #1752.

**Comment by busunkim96** [comment]
This seems to just capture the owlbot changes now, merging.

**Pull request merged**